### PR TITLE
[res] add tflite recipes for circle-opselector-test

### PR DESCRIPTION
**Pull request by parkeunsang**
Related PR about `circle-opselector-test` is #7765
The file name's format is {origin_recipe_name}_Selected_00
ex) Part_Sqrt_Rsqrt_002_Selected_01 consists of partial nodes from Part_Sqrt_Rsqrt_002.

### CODE REVIEW

File: res/TensorFlowLiteRecipes/Part_Sqrt_Rsqrt_002_Selected_01/test.recipe [hunk 1]
```diff
@@ -0,0 +1,37 @@
+operand {
```
Comment (seanshpark) [inline res/TensorFlowLiteRecipes/Part_Sqrt_Rsqrt_002_Selected_01/test.recipe:1]: `Part_xxx` recipes are for circle-partitioner so please use other prefix for opselector like `OpSel_xxx`.

**Comment by seanshpark** [comment]
There is no strict limit of lines of recipe files. But each recipe has some reason of testing.
If you have added some with no particular reason of testing, it would be better to shrink to test that feature or condition of opselecor.

To be more specific, please write down the reason of needs, that is the condition or model's network configuration you want to test. If the the recipe has redundant nodes for that particular test, then you can remove them.
If you share what you wrote, that will help me review this PR.

**Comment by seanshpark** [comment]
@parkeunsang , please make a mew branch from current master and add changes and post a PR.
I don't think current commit message will be clean cause of lots of commits and commit messages.

**Comment by parkeunsang** [comment]
@seanshpark 
I understood. I will send a PR again after excluding unnecessary parts.

**Pull request closed without merging**